### PR TITLE
[SIG-3609] Radio focus 

### PR DESCRIPTION
**Pull request by aditudorache**
This PR sets a custom focus on the radio because on Firefox the browser default focus is not visible on Mac

### CODE REVIEW

File: src/components/RadioButton/index.tsx [hunk 1]
```diff
@@ -22,7 +32,7 @@ interface RadioButtonProps extends InputHTMLAttributes<HTMLInputElement> {
 
 const RadioButton: FunctionComponent<RadioButtonProps> = ({ className, ...props }) => (
   <Wrapper className={className}>
-    <AscRadio {...props} />
+    <StyledRadio {...props} data-test="test" />
```
Comment (MauritsR) [inline src/components/RadioButton/index.tsx:35]: data-test?

**Review by MauritsR — CHANGES_REQUESTED** [review]
Installing the new asc-ui fails. Can you synchronize with develop and try again? Might be that npm@7 is throwing a wrench into it.

File: src/components/RadioButton/index.tsx (hunk 1 above)
Comment (aditudorache) [reply src/components/RadioButton/index.tsx]: Good catch! Thanks!

**Comment by MauritsR** [comment]
Screenshot 2021-04-09 at 12 15 42
I get the result shown above, a mix of the native focus ring and a new black focus ring. Shouldn't the native ring be shown, just more visible? 

This document also describes using the default focus state: [design system - focus state](https://designsystem.amsterdam.nl/7awj1hc9f/p/17684a-focus-state)

**Comment by aditudorache** [comment]
> Screenshot 2021-04-09 at 12 15 42
> I get the result shown above, a mix of the native focus ring and a new black focus ring. Shouldn't the native ring be shown, just more visible?
> 
> This document also describes using the default focus state: [design system - focus state](https://designsystem.amsterdam.nl/7awj1hc9f/p/17684a-focus-state)

It would be the best solution but I don't think is possible. The way how the native focus is shown is the browser responsibility. I couldn't find a way to do that in the application, do you have any suggestion?

**Review by MauritsR — APPROVED** [review]
Looks good, but can you install the packages using npm@7 and recommit the package-lock.json?

**Pull request merged**